server/src/main/java/org/apache/druid/server/coordinator/rules/LoadRule.java

```diff
@@ -119,6 +119,34 @@ public void updateUnderReplicated(
     });
   }
 
+  @Override
+  public void updateUnderReplicatedWithClusterView(
```

Comment:
Can you add an implementation for `BroadcastDistributionRule` as well? I think the implementation there could just call `updateUnderReplicated` and ignore the `cluster` parameter, since it's already looking at the available servers in the cluster.

Reply:
Added.